woocommerce/woocommerce-gutenberg-products-block
# Migrate Cart and Checkout Pages to the Template Editor when using a FSE theme

Adds a mechanism to migrate pages to the template editor by:

1. Taking default content for the template from the page
2. Redirecting the edit page link to the template

Default content is generated from the page content and the page title. This means if you're using a shortcode, or a block, on your cart/checkout page, it will automatically copy across to the FSE template.

Fixes #9334

### Testing



1. Ensure you're running a FSE theme and have cart/checkout pages already setup. Before using this, edit the page content to add something custom/unique e.g. some text, so you can identify the content later.
2. Check out this PR
3. Go to Appearance > Editor > Templates and view the Cart and Checkout templates. They should contain your original page content (unless already customised, in which case you can reset customisations to see the page content)
4. Go to Pages  and edit a Checkout or Cart page. You should be taken to the editor instead.
